### PR TITLE
Stock data

### PR DESCRIPTION
**Pull request by daiterranova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -768,3 +768,108 @@ Now, we can uncomment the line inside of the **options** object:
 const options = {
         colors: [color]
 ```
+
+### Stock Data component
+
+In the `StockDetailPage`, besides the stock chart, we want to show additional information of the selected stock.
```
Comment (reynico) [inline README.md]:
```suggestion
In the `StockDetailPage`, besides the stock chart, we want to show additional information on the selected stock.
```

File: README.md [hunk 2]
```diff
@@ -768,3 +768,108 @@ Now, we can uncomment the line inside of the **options** object:
...
+                            <span className="fw-bold">url: </span>
+                            <a href={stockData.weburl}>{stockData.weburl}</a>
+                        </div>
+                    </div>
+                </div>
+```
+
+Bear in mind that this table is shown when there's data (that means when `stockData` is true).
+
+Following, import the component on the `StockDetailPage` component and returned it like this:
```
Comment (reynico) [inline README.md]:
```suggestion
Following, import the component on the `StockDetailPage` component and return it like this:
```

File: src/images/trading.jpg
Comment (reynico) [inline src/images/trading.jpg:1]: this logo is HUGE. I'd suggest using a resized version of it.

here's a resized one:
trading

**Review by reynico — CHANGES_REQUESTED** [review]

File: src/images/trading.jpg
Comment (daiterranova) [reply src/images/trading.jpg:1]: changed

**Review by reynico — APPROVED** [review]
lgtm!

**Pull request merged**